tests/test_client.py

```diff
@@ -60,7 +60,7 @@ def test_client_authentication_url(api_client, api_url):
             ('state', ''),
             ('redirect_uri', '/redirect'),
             ('response_type', 'code'),
-            ('client_id', 'None'),
+            ('client_id', ''),
```

Comment:
I don't know if you consider this a bug or a feature, but it's a change to call out. The `url_concat()` function would transform `None` values into the string "None". URLObject is more intelligent than that.